src-srv/utils/extensions/auth.ts

```diff
@@ -1,40 +1,27 @@
 import {
-  type onStatelessPayload,
   type Extension,
   type onAuthenticatePayload
 } from '@hocuspocus/server'
 import { type JWTPayload, decodeJwt } from 'jose'
-import { type Repository } from '../Repository.js'
-import { type StatelessAuth, StatelessType, parseStateless } from '@/shared/stateless.js'
-
-interface Configuration {
-  validateToken: Repository['validateToken']
-}
 
 export class Auth implements Extension {
-  configuration: Configuration
-
-  constructor(configuration: Configuration) {
-    this.configuration = { ...configuration }
-  }
-
   async onAuthenticate({ token }: onAuthenticatePayload): Promise<{
     token: string
     user: JWTPayload
   }> {
-    try {
+    /* try {
       await this.configuration.validateToken(token)
     } catch (ex) {
       throw new Error('Could not authenticate', { cause: ex })
-    }
+    } */
 
     return {
       token,
       user: { ...decodeJwt(token) }
     }
   }
 
-  async onStateless({ payload, connection }: onStatelessPayload): Promise<void> {
+  /* async onStateless({ payload, connection }: onStatelessPayload): Promise<void> {
```

Comment:
Is this a leftover commented out code or WiP that should be there?

Reply:
Yeah, it should be reimplemented... I need to take a look at it right afterwards